packages/google_identity_services_web/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## NEXT
+## 0.3.1+5
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* Clean up documentation of callbacks in `CodeClientConfig`,
```

Comment:
Cleans

The PR checklist linked to a CHANGELOG style guide for this repo's packages.